src/Facade/Services/SuppliersService.cs

```diff
@@ -1,24 +1,77 @@
-﻿namespace Linn.Stores.Facade.Services
+﻿//namespace Linn.Stores.Facade.Services
```

Comment:
not sure what's going on here but I wouldn't let this commented out code into master

Reply:
Oops yeah I'll take that out, wasn't going to pr it like that last night then put it up today without checking